src/main/java/org/skriptlang/skript/bukkit/itemcomponents/ItemComponentModule.java

```diff
@@ -0,0 +1,16 @@
+package org.skriptlang.skript.bukkit.itemcomponents;
+
+import ch.njol.skript.Skript;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableModule;
+
+import java.io.IOException;
+
+public class ItemComponentModule {
```

Comment:
I would consider making this an actual module, even if not using the new registration for syntax